### DISCUSSION TITLE
Harmony results loses one dimension

### DISCUSSION THREAD

**Discussion by Zepeng-Mu** [GreenleafLab/ArchR#802] [Questions / Documentation] [answered]
Hi, I noticed a point which I don't quite understand running Harmony. I used `dimsToUse = 1:20` in `addHarmony` function, and the output matrix only has 19 columns. However, I can then use `dimsToUse = 1:20` in `addCluster` without any problem. I'm wondering whether it's always the case that Harmony output loses one dimension?

Thanks!

**Comment by rcorces** [comment, marked as answer]
probably related to `corCutoff` in `addHarmony()`
https://www.archrproject.com/reference/addHarmony.html

**Reply by Zepeng-Mu** [reply]
I see. Indeed there was message for removing one dimension that is correlated with number of fragments.
Thanks!
